[2024-07-03T09:57:23.406+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: reports_workflow.id_task_get_emails scheduled__2024-07-02T07:50:00+00:00 [queued]>
[2024-07-03T09:57:23.422+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: reports_workflow.id_task_get_emails scheduled__2024-07-02T07:50:00+00:00 [queued]>
[2024-07-03T09:57:23.422+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2024-07-03T09:57:23.452+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): id_task_get_emails> on 2024-07-02 07:50:00+00:00
[2024-07-03T09:57:23.456+0000] {standard_task_runner.py:57} INFO - Started process 20521 to run task
[2024-07-03T09:57:23.470+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'reports_workflow', 'id_task_get_emails', 'scheduled__2024-07-02T07:50:00+00:00', '--job-id', '339', '--raw', '--subdir', 'DAGS_FOLDER/dag_email_robot.py', '--cfg-path', '/tmp/tmp_klwmorx']
[2024-07-03T09:57:23.472+0000] {standard_task_runner.py:85} INFO - Job 339: Subtask id_task_get_emails
[2024-07-03T09:57:23.555+0000] {task_command.py:415} INFO - Running <TaskInstance: reports_workflow.id_task_get_emails scheduled__2024-07-02T07:50:00+00:00 [running]> on host 6f4ef5488251
[2024-07-03T09:57:23.673+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='FIRDAVS' AIRFLOW_CTX_DAG_ID='reports_workflow' AIRFLOW_CTX_TASK_ID='id_task_get_emails' AIRFLOW_CTX_EXECUTION_DATE='2024-07-02T07:50:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-07-02T07:50:00+00:00'
[2024-07-03T09:57:23.674+0000] {logging_mixin.py:151} INFO - path of config file:  /opt/***
[2024-07-03T09:57:23.675+0000] {taskinstance.py:1943} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py", line 192, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py", line 209, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/dag_email_robot.py", line 22, in get_emails
    email_conf = json.load (open(path / ("config/email_conf.json")))
NameError: name 'json' is not defined
[2024-07-03T09:57:23.704+0000] {taskinstance.py:1400} INFO - Marking task as FAILED. dag_id=reports_workflow, task_id=id_task_get_emails, execution_date=20240702T075000, start_date=20240703T095723, end_date=20240703T095723
[2024-07-03T09:57:23.732+0000] {standard_task_runner.py:104} ERROR - Failed to execute job 339 for task id_task_get_emails (name 'json' is not defined; 20521)
[2024-07-03T09:57:23.796+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 1
[2024-07-03T09:57:23.851+0000] {taskinstance.py:2784} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-07-03T12:04:53.757+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: reports_workflow.id_task_get_emails scheduled__2024-07-02T07:50:00+00:00 [queued]>
[2024-07-03T12:04:53.776+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: reports_workflow.id_task_get_emails scheduled__2024-07-02T07:50:00+00:00 [queued]>
[2024-07-03T12:04:53.776+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2024-07-03T12:04:53.810+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): id_task_get_emails> on 2024-07-02 07:50:00+00:00
[2024-07-03T12:04:53.815+0000] {standard_task_runner.py:57} INFO - Started process 23335 to run task
[2024-07-03T12:04:53.829+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'reports_workflow', 'id_task_get_emails', 'scheduled__2024-07-02T07:50:00+00:00', '--job-id', '780', '--raw', '--subdir', 'DAGS_FOLDER/dag_email_robot.py', '--cfg-path', '/tmp/tmpu2vn4c9e']
[2024-07-03T12:04:53.831+0000] {standard_task_runner.py:85} INFO - Job 780: Subtask id_task_get_emails
[2024-07-03T12:04:53.920+0000] {task_command.py:415} INFO - Running <TaskInstance: reports_workflow.id_task_get_emails scheduled__2024-07-02T07:50:00+00:00 [running]> on host 6f4ef5488251
[2024-07-03T12:04:53.998+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='FIRDAVS' AIRFLOW_CTX_DAG_ID='reports_workflow' AIRFLOW_CTX_TASK_ID='id_task_get_emails' AIRFLOW_CTX_EXECUTION_DATE='2024-07-02T07:50:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-07-02T07:50:00+00:00'
[2024-07-03T12:04:53.999+0000] {logging_mixin.py:151} INFO - path of config file:  /opt/***
[2024-07-03T12:04:53.999+0000] {logging_mixin.py:151} INFO - #>2024-07-03 12:04:53.999750_User@StatDep: EmailRobot in screated.
[2024-07-03T12:04:53.999+0000] {logging_mixin.py:151} INFO - #>2024-07-03 12:04:53.999917_User@StatDep: get_for_recent_emails is started...
[2024-07-03T12:04:54.000+0000] {logging_mixin.py:151} INFO - #>2024-07-03 12:04:54.000029_User@StatDep: pop3_fetch_emails isstarted...
[2024-07-03T12:04:54.415+0000] {logging_mixin.py:151} INFO -    b'+OK'
[2024-07-03T12:04:54.862+0000] {logging_mixin.py:151} INFO -    mailbox have 1 emails and the robot can process 10
[2024-07-03T12:04:55.321+0000] {logging_mixin.py:151} INFO -    #>2024-07-03 12:04:55.321527_conneciton to db is set.
[2024-07-03T12:04:55.331+0000] {taskinstance.py:1943} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py", line 192, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py", line 209, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/dag_email_robot.py", line 25, in get_emails
    robot.fetch_emails()
  File "/opt/airflow/dags/emailRobot.py", line 62, in fetch_emails
    return self.pop3_fetch_emails()
  File "/opt/airflow/dags/emailRobot.py", line 156, in pop3_fetch_emails
    cursor.execute (query, insert_values)
psycopg2.errors.UndefinedTable: relation "sma_stat_dep.tbl_file_upload" does not exist
LINE 1: ...) AS (INSERT INTO                                 sma_stat_d...
                                                             ^

[2024-07-03T12:04:55.350+0000] {taskinstance.py:1400} INFO - Marking task as FAILED. dag_id=reports_workflow, task_id=id_task_get_emails, execution_date=20240702T075000, start_date=20240703T120453, end_date=20240703T120455
[2024-07-03T12:04:55.366+0000] {standard_task_runner.py:104} ERROR - Failed to execute job 780 for task id_task_get_emails (relation "sma_stat_dep.tbl_file_upload" does not exist
LINE 1: ...) AS (INSERT INTO                                 sma_stat_d...
                                                             ^
; 23335)
[2024-07-03T12:04:55.413+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 1
[2024-07-03T12:04:55.450+0000] {taskinstance.py:2784} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-07-03T12:22:42.313+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: reports_workflow.id_task_get_emails scheduled__2024-07-02T07:50:00+00:00 [queued]>
[2024-07-03T12:22:42.326+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: reports_workflow.id_task_get_emails scheduled__2024-07-02T07:50:00+00:00 [queued]>
[2024-07-03T12:22:42.326+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2024-07-03T12:22:42.346+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): id_task_get_emails> on 2024-07-02 07:50:00+00:00
[2024-07-03T12:22:42.361+0000] {standard_task_runner.py:57} INFO - Started process 24184 to run task
[2024-07-03T12:22:42.371+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'reports_workflow', 'id_task_get_emails', 'scheduled__2024-07-02T07:50:00+00:00', '--job-id', '1068', '--raw', '--subdir', 'DAGS_FOLDER/dag_email_robot.py', '--cfg-path', '/tmp/tmp_vayvxq_']
[2024-07-03T12:22:42.372+0000] {standard_task_runner.py:85} INFO - Job 1068: Subtask id_task_get_emails
[2024-07-03T12:22:42.468+0000] {task_command.py:415} INFO - Running <TaskInstance: reports_workflow.id_task_get_emails scheduled__2024-07-02T07:50:00+00:00 [running]> on host 6f4ef5488251
[2024-07-03T12:22:42.591+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='FIRDAVS' AIRFLOW_CTX_DAG_ID='reports_workflow' AIRFLOW_CTX_TASK_ID='id_task_get_emails' AIRFLOW_CTX_EXECUTION_DATE='2024-07-02T07:50:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-07-02T07:50:00+00:00'
[2024-07-03T12:22:42.592+0000] {logging_mixin.py:151} INFO - path of config file:  /opt/***
[2024-07-03T12:22:42.593+0000] {logging_mixin.py:151} INFO - #>2024-07-03 12:22:42.593579_User@StatDep: EmailRobot in screated.
[2024-07-03T12:22:42.593+0000] {logging_mixin.py:151} INFO - #>2024-07-03 12:22:42.593813_User@StatDep: get_for_recent_emails is started...
[2024-07-03T12:22:42.593+0000] {logging_mixin.py:151} INFO - #>2024-07-03 12:22:42.593953_User@StatDep: pop3_fetch_emails isstarted...
[2024-07-03T12:22:42.996+0000] {logging_mixin.py:151} INFO -    b'+OK'
[2024-07-03T12:22:43.318+0000] {logging_mixin.py:151} INFO -    mailbox have 0 emails and the robot can process 10
[2024-07-03T12:22:43.318+0000] {logging_mixin.py:151} INFO - #>2024-07-03 12:22:43.318873_User@StatDep: no emails to process, exiting pop3_fetch_emails()...
[2024-07-03T12:22:43.319+0000] {python.py:194} INFO - Done. Returned value was: True
[2024-07-03T12:22:43.359+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=reports_workflow, task_id=id_task_get_emails, execution_date=20240702T075000, start_date=20240703T122242, end_date=20240703T122243
[2024-07-03T12:22:43.396+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2024-07-03T12:22:43.430+0000] {taskinstance.py:2784} INFO - 1 downstream tasks scheduled from follow-on schedule check
